src/ord.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const {tagged} = require('daggy');
+const {empty, of, concat, equals} = require('fantasy-land');
+
+const Ord = M => {
+
+    const _Ord = tagged('x');
+
+    _Ord[of] = _Ord;
+    _Ord[Ord.min] = M[Ord.min];
+    _Ord[Ord.max] = M[Ord.max];
+
+    _Ord.prototype.compare = function(y) {
+      return M.compare(this.x, y.x);
+    };
+    _Ord.prototype[equals] = function(y) {
```

Comment:
Because this is the new fantasyland naming scheme, you can read all of this
if you want https://github.com/fantasyland/fantasy-land/issues/92

On Sun, 21 Feb 2016, 09:51 Tobias Pflug <notifications@github.com> wrote:

> In src/ord.js
> <https://github.com/fantasyland/fantasy-monoids/pull/10#discussion_r53564307>
> :
>
> > +
> > +const {tagged} = require('daggy');
> > +const {empty, of, concat, equals} = require('fantasy-land');
> > +
> > +const Ord = M => {
> > +
> > +    const _Ord = tagged('x');
> > +
> > +    _Ord[of] = _Ord;
> > +    _Ord[Ord.min] = M[Ord.min];
> > +    _Ord[Ord.max] = M[Ord.max];
> > +
> > +    _Ord.prototype.compare = function(y) {
> > +      return M.compare(this.x, y.x);
> > +    };
> > +    _Ord.prototype[equals] = function(y) {
>
> Why prototype[equals] instead of just prototype.equals ? just looks weird
> to me ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fantasyland/fantasy-monoids/pull/10/files#r53564307>.
>